core/lib/common/worker.js

```diff
@@ -458,7 +458,7 @@ module.exports = class Worker {
 			`${dutPort}:${ip}:${dutPort}`,
 			`-p`,
 			workerPort,
-			`root@127.0.0.1`,
+			`${this.username}@127.0.0.1`,
```

Comment:
```suggestion
			`${this.username ? this.username : "root"}@127.0.0.1`,
```

What about dev mode then? Is username default to root in other situations? On L87 when it's defined, it's `this.username = username`

Reply:
Might be better to use my suggestion on Line 87 only where username is initialized

Reply:
@vipulgupta2048 , we can't default it to root, it is required to be defined, and also used elsewhere. 

SSH auth to the worker will always be via BC ssh keys auth, it works if the worker is dev or prod mode that way